File: 장승민/CSharp/210208/210208_carClass/210208_carClass/Car.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _210208_carClass
{
    class Car
    {
        public const string STORE = "동대구 영업소";
        private string maker;
        private string color;
        private string model;
        private string price;

        public Car()
        {
            maker = "기아";
            color = "블랙";
            model = "K7";
            price = "사천만원";
        }
        public Car(string maker, string color, string model, string price)
        {
            this.maker = maker;
            this.color = color;
            this.model = model;
            this.price = price;
        }

        public string Maker { get => maker; set => maker = value; }
        public string Color { get => color; set => color = value; }
        public string Model { get => model; set => model = value; }
        public string Price { get => price; set => price = value; }



        /* public string Maker
         {
             get { return maker; }
             set { maker = value; }
         }

         public string Color
         {
             get { return color; }
             set { color = value; }
         }
         public string Model
         {
             get { return model; }
             set { model = value; }
         }
         public string Price
         {
             get { return price; }
             set { price = value; }
         }*/

        public void showCarInfo()
        {
            Console.WriteLine("제조사: " + maker);
            Console.WriteLine("색상: " + color);
            Console.WriteLine("이름: " + model);
            Console.WriteLine("가격: " + price);
            Console.WriteLine(STORE);
            Console.WriteLine("=================");
        }

    }
}

File: 장승민/Java/210112_classPractice1/src/com/kb/carapp/car/Car.java
package com.kb.carapp.car;

public class Car {
	private String maker;
	private String color;
	private String model;
	private String money;
	
	public static String store = "동대구 영업소";
	
	public Car() {
		maker = "기아";
		color="블랙";
		model = "K7";
		money = "4천만원";
	}
	
	

	public String getMaker() {
		return maker;
	}



	public void setMaker(String maker) {
		this.maker = maker;
	}



	public String getColor() {
		return color;
	}



	public void setColor(String color) {
		this.color = color;
	}



	public String getModel() {
		return model;
	}



	public void setModel(String model) {
		this.model = model;
	}



	public String getMoney() {
		return money;
	}



	public void setMoney(String money) {
		this.money = money;
	}



	public Car(String maker, String color, String model, String money) {
		super();
		this.maker = maker;
		this.color = color;
		this.model = model;
		this.money = money;
	}
	
	public void showCarInfo() {
		System.out.println("제조사 : " + maker);
		System.out.println("색상 : " + color);
		System.out.println("모델 이름 : " + model);
		System.out.println("가격 : " + money);
		System.out.println(store);
		System.out.println("--------------------");
	}
	
}
